rx/concurrency/schedulerbase.py

```diff
@@ -6,15 +7,94 @@
 from rx.internal.basic import default_now
 from rx.internal.constants import DELTA_ZERO, UTC_ZERO
 
+
 class SchedulerBase(typing.Scheduler):
     """Provides a set of static properties to access commonly used
     schedulers.
     """
 
+    @property
+    def now(self) -> datetime:
+        """Represents a notion of time for this scheduler. Tasks being
+        scheduled on a scheduler will adhere to the time denoted by this
+        property.
+
+        Returns:
+             The scheduler's current time, as a datetime instance.
+         """
+
+        return default_now()
+
+    @abstractmethod
+    def schedule(self,
```

Comment:
I'm just curious, is it worth to re-declare `schedule`, `schedule_relative` & `schedule_absolute` as abstract methods while these are already declared in `typing.Scheduler` ?

Reply:
Yeah, maybe not strictly necessary... Although it might help to keep the generated docs and `help()` complete.

I guess I included this mainly because of an idea I have that I’m playing around with, which would simplify all of the classes that extend `SchedulerBase` by putting some often-repeated logic (schedule methods being defined in terms of one another) in the base class.

Having these here at this point would make that second step a lot easier / readable.

Reply:
cool :+1: